website_node_branded/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+website_node_branded
+====================
+
+Once installed, you can use the t-ignore-branding attribute on a <t> node to control if node is branded or qweb-active (bears a
+@data-oe-model or a @t-* *which is not t-field* as t-field does not section out views)
```

Comment:
@Yajo In Odoo if your snippet insert a '<t>'  element, the element will be removed when you save the page an not interpreted py the QWeb processor when the page is rendered. This technical module allow you to bypass this limitation. The breadcrum snippet is basic and only insert a '<t t-call=...' element into the page https://github.com/acsone/website/blob/website_snippet_breadcrumb-jne/website_snippet_breadcrumb/static/src/js/website_snippet_breadcrumb.editor.js#L37
The credits of this Idea comes from Therp BV (https://github.com/OCA/website/pull/129)